src/ui/components/MetadataCard/index.js

```diff
@@ -1,22 +1,23 @@
 /* @flow */
+import invariant from 'invariant';
+import makeClassName from 'classnames';
 import * as React from 'react';
 
 import LoadingText from 'ui/components/LoadingText';
 
 import './styles.scss';
 
 type Props = {|
+  className?: string,
   metadata: Array<Object>,
 |};
 
-const MetadataCard = (props: Props) => {
-  if (!props.metadata) {
-    throw new Error('The metadata property is required');
-  }
+const MetadataCard = ({ className, metadata }: Props) => {
+  invariant(metadata, 'The metadata property is required');
 
   return (
-    <div className="MetadataCard">
-      {props.metadata.map(({ content, title } = {}) => {
+    <div className={makeClassName('MetadataCard', className)}>
+      {metadata.map(({ content, title } = {}) => {
         if (content === undefined) {
           throw new Error('content is required');
```

Comment:
just curious - I know this isn't part of your patch but can/should we use invariant here?

Reply:
Yes, that line pre-dates our usage of invariant. I didn't change it because I think some tests will need updating and I didn't want to make it a bigger patch.

Reply:
I figured I just want to make sure : )